src/components/dashboard/DashboardPage.jsx

```diff
@@ -19,15 +19,24 @@ import CourseEnrollmentFailedAlert, { ENROLLMENT_SOURCE } from '../course/Course
 import { ProgramListingPage } from '../program-progress';
 import PathwayProgressListingPage from '../pathway-progress/PathwayProgressListingPage';
 import { features } from '../../config';
+import { useEnterpriseCuration } from '../search/content-highlights/data';
+import { useLearnerProgramsListData } from '../program-progress/data/hooks';
+import { useInProgressPathwaysData } from '../pathway-progress/data/hooks';
 
 export const LICENCE_ACTIVATION_MESSAGE = 'Your license was successfully activated.';
-
 const DashboardPage = () => {
-  const { enterpriseConfig, authenticatedUser } = useContext(AppContext);
-  const { subscriptionPlan, showExpirationNotifications } = useContext(UserSubsidyContext);
   const { state } = useLocation();
   const history = useHistory();
+  const { enterpriseConfig, authenticatedUser } = useContext(AppContext);
+  const { subscriptionPlan, showExpirationNotifications } = useContext(UserSubsidyContext);
   const [isActivationAlertOpen, , closeActivationAlert] = useToggle(!!state?.activationSuccess);
+  const [learnerProgramsListData, programsFetchError] = useLearnerProgramsListData(enterpriseConfig.uuid);
+  const [pathwayProgressData, pathwayFetchError] = useInProgressPathwaysData(enterpriseConfig.uuid);
```

Comment:
Considering moving these 2 data fetch functionalities into `SubsidyRequestsContextProvider` to be accessible across AuthenticatedUerSubsidyPage's children which include the dashboard page.

Reply:
Additionally, had to rename non-unique `fetchError` to something more recognizable since both API calls are accessible within the same component now. `fetchError` resulted in duplicate const name error.

Reply:
> Considering moving these 2 data fetch functionalities into SubsidyRequestsContextProvider to be accessible across AuthenticatedUerSubsidyPage's children which include the dashboard page.

I'm not sure it makes sense to put data about learners' programs/pathways data into `SubsidyRequestsContextProvider`, which is intended to be specific to subsidy requests (Browse & Request). 

That said, I could see these data being useful across the application beyond just the dashboard page so I would be in support moving these some higher up in the component tree; I just don't know if we have a existing context provider that would make sense for it vs. having to create a new context provider.

Reply:
Leaving open for reference to techdebt